includes/admin/class-wcmp-export.php

```diff
@@ -1033,6 +1035,27 @@ public static function getPackageTypeAsString($packageType): string
         return $packageType ?? AbstractConsignment::DEFAULT_PACKAGE_TYPE_NAME;
     }
 
+    /**
+     * @param WC_Order $order
+     * @param string   $packageType
+     *
+     * @return string
+     *
+     * @throws Exception
+     */
+    public function getAllowedPackageType(WC_Order $order, string $packageType): string
+    {
+        $shipping_country = WCX_Order::get_prop($order, "shipping_country");
+        $mailbox          = AbstractConsignment::PACKAGE_TYPE_MAILBOX_NAME === $packageType;
+        $digitalStamp     = AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP_NAME === $packageType;
+
+        if (AbstractConsignment::CC_NL != $shipping_country && ($mailbox || $digitalStamp)) {
+            return AbstractConsignment::DEFAULT_PACKAGE_TYPE_NAME;
```

Comment:
Omdat je al `$packageType` gaat returnen is mijn voorkeur om maar 1 return point te hebben, en dus dat je `$packageType = AbstractConsignment::DEFAULT_PACKAGE_TYPE_NAME;`doet

Reply:
done